WeakAurasOptions/OptionsFrames/TexturePicker.lua

```diff
@@ -243,13 +243,19 @@ local function ConstructTexturePicker(frame)
           width = atlasInfo.width
           height = atlasInfo.height
         end
+      else
+        flipbookInfo = OptionsPrivate.GetFlipbookTileSize(pickedwidget.texture.path)
+        if flipbookInfo then
+          width = flipbookInfo.tileWidth
+          height = flipbookInfo.tileHeight
+        end
       end
     end
 
     wipe(group.selectedTextures)
     group.selectedTextures[texturePath] = true
 
-    SetAll(self.baseObject, self.path, self.properties.texture, texturePath, width, height, self.adjustSize)
+    SetAll(self.baseObject, self.path, self.properties.texture, texturePath, width, height, flipbookInfo and true or self.adjustSize)
```

Comment:
I think this change here is the wrong place to add that. Keep this line as is, and instead modify the calls to OpenTexturePicker to pass in true as the last parameter.